bootstrap_cfn/fab_tasks.py

```diff
@@ -622,7 +621,8 @@ def get_cloudformation_tags():
     """
     return {
         "Env": env.environment,
-        "Application": env.application
+        "Application": env.application,
+        "Bootstrap-cfn Version": bootstrap_cfn.__version__
```

Comment:
Since this is a hash, the order of the key-value pairs don't matter. It'd be neater to put the `Bootstrap-cfn Version` tag before `Application` such that only 1 line appears in the diff output.